frontend/hub/access/common/HubRoleWizardSteps/HubSelectTeamsStep.tsx

```diff
@@ -0,0 +1,57 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ITableColumn, TextCell } from '../../../../../framework';
+import { PageMultiSelectList } from '../../../../../framework/PageTable/PageMultiSelectList';
+import { Title } from '@patternfly/react-core';
+import styled from 'styled-components';
+import { useHubTeamFilters } from '../hooks/useHubTeamFilters';
+import { HubTeam } from '../../../interfaces/expanded/HubTeam';
+import { hubAPI } from '../../../common/api/formatPath';
+import { useHubMultiSelectListView } from '../../../common/useHubMultiSelectListView';
+
+const StyledTitle = styled(Title)`
+  margin-bottom: 1rem;
+`;
+
+export function HubSelectTeamsStep(props: { descriptionForTeamsSelection?: string }) {
+  const toolbarFilters = useHubTeamFilters();
+  const { descriptionForTeamsSelection } = props;
+  const { t } = useTranslation();
+
+  const tableColumns: ITableColumn<HubTeam>[] = useMemo(() => {
+    return [
+      {
+        header: t('Name'),
+        cell: (team: HubTeam) => <TextCell text={team.name} />,
+        card: 'name',
+        list: 'name',
+        sort: 'name',
+        maxWidth: 200,
+      },
+    ];
+  }, [t]);
+
+  const view = useHubMultiSelectListView<HubTeam>(
+    {
+      url: hubAPI`/_ui/v2/teams/`,
```

Comment:
Is this a new api or a typo? `/_ui/v2/` is not something we used before

Reply:
@himdel  This is a new API. Available on the RBAC branch: https://github.com/ansible/galaxy_ng/pull/2202 that we are testing these changes with.